src/popup/components/ProtocolSpecificView.vue

```diff
@@ -54,29 +63,38 @@ export default defineComponent({
   },
   setup(props) {
     const store = useStore();
-    const { activeAccount } = useAccounts({ store });
-    const { activeNetwork } = useNetworks();
     const { params, meta } = useRoute();
+
+    const componentToDisplay = ref();
+
+    const { activeNetwork } = useNetworks();
+    const { activeAccount } = useAccounts({ store });
+
     const ownerProtocol = detectProtocolByOwner(
       activeNetwork.value.type,
       params.transactionOwner as string,
     );
-    const importViewComponent = views[
-      meta.isMultisig
-        ? PROTOCOL_AETERNITY
-        : (ownerProtocol || activeAccount.value.protocol)
-    ]?.[props.viewComponentName];
 
-    if (!importViewComponent) {
-      Logger.write({
-        message: `Failed to access "${props.viewComponentName}" component for the "${activeAccount.value.protocol}" protocol`,
-        type: 'vue-error',
-      });
-    }
+    onMounted(async () => {
```

Comment:
This should not happen in the `onMounted` hook as it adds tiny delay to the rendering of the loaded component.

Reply:
Otherwise it's not having the access to the up to date store and crashing